schedule/configdb.py

```diff
@@ -80,15 +80,33 @@ def get_telescope_tuples(self, include_blank=False):
             telescopes.append(('', ''))
         return telescopes
 
-    def telescope_exists(self, site_code, enclosure_code, telescope_code):
+    def get_instrument_type_tuples(self, include_blank=False):
+        instrument_types = set()
+        for site in self.get_site_data():
+            for enclosure in site['enclosure_set']:
+                for telescope in enclosure['telescope_set']:
+                    for instrument in telescope['instrument_set']:
+                        instrument_types.add(instrument['instrument_type']['code'].upper())
```

Comment:
Does filtering for the upper-ed instrument type work? If the instrument type of the downtime is just the string taken from configdb, which might not be all uppercase, then I don't think it would.

Reply:
Yeah I guess I agree, I'll remove the upper()ing